compiler/control/CompileMethod.cpp

```diff
@@ -312,6 +312,10 @@ compileMethodFromDetails(
             compiler.setDebug(createDebugObject(&compiler));
             }
 
+#ifdef VERIFY_IL
```

Comment:
1. There is no "good" default for the option, unless you make it depend on the debug option (and even then, it could just use upon debug or prod with assumes instead of a new option).
2. Currently, this is only implemented in the Test compiler, making it not make sense to either enable in a different project, or disable it in the Test compiler (since it would just stop some tests from working).

If verifiers are used in other projects in the future, this will need to be revisited. In particular, I feel verifiers would be implemented project-wide, not method-wide, meaning this code would need to be removed anyway.